ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py

```diff
@@ -333,7 +333,7 @@ def query_table_names_and_types(
         to get the names and pass the Regular type.
 
         This is useful for sources where we need fine-grained
-        logic on how to handle table types, e.g., external, foreign,...
+        logic on how to handle table types, e.g., external, foreign,...processor
```

Comment:
maybe a typo

Reply:
my laptop has been randomly pasting random shit 😮‍💨 let me fix this, thanks